Ecierge.Uno.Controls/Behaviors/HorizontalListViewSelectionIndicatorBehavior.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.Xaml.Interactivity;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+using System.Linq;
+using CommunityToolkit.WinUI;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Windows.UI;
+
+namespace Ecierge.Uno.Controls.Behaviors
+{
+    public class HorizontalListViewSelectionIndicatorBehavior : Behavior<ListView>
+    {
+        private ListViewItem? previousSelectedItem;
+
```

Comment:
Maybe declare a style here and then just apply the style?
```suggestion

private readonly Style indicatorStyle = new Style() {
    TargetType = typeof(Border),
    Setters = [
        new Setter { Property = "BorderThickness", Value = new Thickness(3) }
    ]
};
```